src/Mp3/Mp3Tagger.php

```diff
@@ -22,40 +21,54 @@
 
 class Mp3Tagger implements TaggerInterface
 {
+    private $logName = 'MP3_TAGGER_LOG';
+    private $loggingEnabled = false;
+
     public function __construct(
         getID3 $getid3,
         getid3_writetags $writeTags,
         ProcessInterface $process,
         LoggerInterface $logger,
+        HandlerInterface $handler,
         FilenameSanitizerInterface $filenameSanitizer,
         Mid3v2CommandInterface $command,
         ValidatorInterface $validator
```

Comment:
I think we are getting real close to violating the SRP here with so many injected dependencies. I personally try and keep them to a maximum of 4 total. Normally any more than that means that the class likely needs to be refactored. If a single class has so many dependencies its more than likely doing to much on its own violating the SRP (*Single responsibility Principle*)

Reply:
This is a well-founded concern.

Let's think on it and determine if there's a more logical way to structure this and similar classes.

Similar opinions: https://stackoverflow.com/questions/953407/constructor-injection-how-many-dependencies-is-too-many

Reply:
I have some ideas as to how we can solve this problem and will aim to push them up this evening.

Reply:
I just pushed a few commits that reduce the number of injected dependencies. It would be easy enough to consolidate the `LoggerInterface` and `HandlerInterface` into a single dependency, too, in which case there would be 4, total.

Reply:
I'm content to merge this and create a separate issue for better adhering to SRP in a subsequent Issue.

Reply:
I bit the bullet and reduced the dependencies to no more than 4 in all instances. Take a look and see what you think.